### PR TITLE
fix: show the kyc status message on the frontend

### PR DESCRIPTION
**Pull request by mhenriette**
# Submit a pull request

- [x] This is not a duplicate of an existing pull request.
- [x] No existing features have been broken without good reason.
- [x] Your commit messages are detailed
- [x] The code style guideline have been followed.
- [ ] Documentation has been updated to reflect your changes.
- [ ] Tests have been added or updated to reflect your changes.
- [ ] All tests pass.

---

Replace any ":question:" below with information about your pull request.

## Pull Request Details

This pull request shows the KYC verification message on the frontend according to the status

## Breaking Changes

none

### CODE REVIEW

File: src/store/feature/kyc.slice.ts [hunk 1]
```diff
@@ -6,6 +6,12 @@ import api from "@/config/api";
 import { fetchUser } from "../services/user.service";
 import snsWebSdk from "@sumsub/websdk";
 
+export enum KYCSTATUS {
```
Comment (serapieTuyishime) [inline src/store/feature/kyc.slice.ts:9]: Its better that all types and enums are moved to the types directory

File: src/components/popups/KYCVerification.tsx [hunk 2]
```diff
@@ -32,7 +33,10 @@ export default function KYCVerification({ onCompleted }: KYCVerificationProps) {
 
   const verificationData = useSelector((state) => state.sumsubVerification);
 
-  const { showModal, completed, completedText, description, verifying, completedActionText, actionText, loading, title } = verificationData;
+  const { showModal, completed, verifying, completedActionText, actionText, loading, title } = verificationData;
+  const user = useSelector((state) => state.user.data);
+
+  const kycStatus = useMemo(() => user?.kycStatus, [user?.kycStatus]);
```
Comment (serapieTuyishime) [inline src/components/popups/KYCVerification.tsx:39]: Its not necessary to use useMemo for one variable.

File: src/components/popups/KYCVerification.tsx [hunk 3]
```diff
@@ -48,13 +52,20 @@ export default function KYCVerification({ onCompleted }: KYCVerificationProps) {
     triggerCompleteAction();
   };
 
+  const statuMessage = useMemo(() => {
```
Comment (Odile078) [inline src/components/popups/KYCVerification.tsx:55]: there is a typo in this variable naming, should be `statusMessage`

**Review by Odile078 — APPROVED** [review]

File: src/components/ui/Popup.tsx [hunk 4]
```diff
@@ -31,6 +31,15 @@ interface PopupProps {
...
 export default function Popup({ center, onClose, show, children, className = "" }: PopupProps): ReactElement {
+
+  useEffect(() => {
+    if (show) {
+      document.body.style.overflow = "hidden";
+    } else {
+      document.body.style.overflow = "auto";
+    }
+  }, [show]);
+
```
Comment (serapieTuyishime) [inline src/components/ui/Popup.tsx:42]: We have a function for this: `toggleBodyScrolling()`

**Review by serapieTuyishime — APPROVED** [review]

**Review by Emmanuel-umeh — APPROVED** [review]
LGTM

**Review by musayann — APPROVED** [review]

**Pull request merged**